mlflow/xgboost.py

```diff
@@ -130,7 +138,29 @@ def save_model(
                           model. The given example will be converted to a Pandas DataFrame and then
                           serialized to json using the Pandas split-oriented format. Bytes are
                           base64-encoded.
-
+    :param pip_requirements: Overwrites a default set of pip requirements generated automatically
```

Comment:
Can we adjust the phrasing here so that it's similar to `conda_env`? How about:

```
Either an iterable of pip requirement strings (e.g. ``["scikit-learn", "-r requirements.txt"]``) or the string path to a pip requirements file on. the local filesystem (e.g. ``"requirements.txt"``). If provided, this describes the environment this model should be run in. If ``None``, a default list of requirements is inferred from the current software environment.
```

Reply:
Sure!